backend/models/contributions.js

```diff
@@ -0,0 +1,50 @@
+const bcrypt = require('bcrypt-nodejs');
+const User = require(`./users.js`);
+
+module.exports = (sequelize, DataTypes) => {
+  const Contribution = sequelize.define('user', {
+    //bute_id that will be used for individual contributions
+    //TODO: tell postgres to generate unique uuid
+    bute_id: {
+      type: DataTypes.UUID,
+      allowNull: false,
+      unique: true,
+      validate: {
+        notEmpty: true,
+        isUUID: true, //sequelize validation for uuid
+      },
+    },
+    data: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      unique: false,
+      validate: {
+        notEmpty: true,
+      },
+    },
+    // author: {
+    //   type: DataTypes.STRING,
+    //   allowNull: false,
+    //   unique: true,
+    //   validate: {
+    //     notEmpty: true,
+    //     isEmail: true, // sequelize validation
+    //   },
+    // },
+    //the dynamic array linking to contribution table
+    prev_versions: {
```

Comment:
We are abandoning the `prev_versions` with just one previous version. 

The way this will be is that at the very end of the file, you'll have something like

`Contribution.hasOne(Contribution, {as: 'ancestor'});`

See [this](https://github.com/sequelize/sequelize/issues/1937) for more info